contract-message-predicate/tests/harness.rs

```diff
@@ -0,0 +1,121 @@
+use fuel_crypto::Hasher;
+use fuels::contract::script::Script;
+use fuels::prelude::*;
+use fuels::test_helpers::Config;
+use fuels::tx::{AssetId, Contract, Input, Output, Transaction, UtxoId};
+
+async fn get_balance(provider: &Provider, address: Address, asset: AssetId) -> u64 {
+    let balance = provider.get_asset_balance(&address, asset).await.unwrap();
+    balance
+}
+
+#[tokio::test]
+async fn spend_predicate_with_script_constraint() {
```

Comment:
I see a single test here. Should there be an additional negative test which proves some behaviour e.g. if predicate returns false or something goes wrong?

Reply:
I've created an issue #5 to focus on better tests. For now I just want to get the separation of repos finished